src/NextGen/DonationForm/Blocks/DonationFormBlock/app/utilities/postData.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @unreleased
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_request_options
+ */
+export default async function postData(url: string, data: object = {})  {
+  // Default options are marked with *
+  const response = await fetch(url, {
+    method: 'POST', // *GET, POST, PUT, DELETE, etc.
+    mode: 'cors', // no-cors, *cors, same-origin
```

Comment:
Don't many/most of the requests go to the same origin? I'm not sure we really need to hard-code this.

Reply:
Yeah, I think same-origin is fine.  `mode: 'cors'` is actually the default (noted by * in comments) so we would need to mark this as `same-origin`.

Reply:
Changed mode to `same-origin` 0f13c2e4 8e305df3

Let's keep this config with the comments in tact for now. I want to make sure it's setup appropriately as we introduce more gateways.